src/levanter/models/qwen.py

```diff
@@ -219,6 +254,49 @@ def __call__(
         x = self.norm(x)
         return x
 
+    @named_call
+    def decode(
```

Comment:
I noticed that the decode and initial_cache are directly copied from the LlamaTransformer superclass. I think those two methods should be inherited?